change/@fluentui-react-carousel-d45f8c82-f098-4820-9e34-31862a08f3d3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33411&commitId=02d83b56928fb9296f14c7cc182a479f0082e2d7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|
|Avatar Converged.badgeMask.chromium.png|4| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1147| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>